core/base/persistenceDiagramClustering/PersistenceDiagramBarycenter.h

```diff
@@ -450,20 +447,15 @@ namespace ttk {
         }
       }
 
-      if(debugLevel_ > 0) {
-        std::cout << "[PersistenceDiagramBarycenter] Total cost : "
-                  << total_cost << std::endl;
-      }
-      std::stringstream msg;
-      msg << "[PersistenceDiagramBarycenter] processed in "
-          << tm.getElapsedTime() << " s. (" << threadNumber_ << " thread(s))."
-          << std::endl;
-      dMsg(std::cout, msg.str(), timeMsg);
+      printMsg("Total cost : " + std::to_string(total_cost));
+      // std::stringstream msg;
+      // msg << "[PersistenceDiagramBarycenter] processed in "
+      //     << tm.getElapsedTime() << " s. (" << threadNumber_ << "
+      //     thread(s))."
+      //     << std::endl;
+      // dMsg(std::cout, msg.str(), timeMsg);
+      printMsg("", tm.getElapsedTime(), threadNumber_);
```

Comment:
see https://github.com/topology-tool-kit/ttk/blob/dev/core/base/helloWorld/HelloWorld.h (l. 123) for an example of time logging

Reply:
right, my bad